src/cluster.ts

```diff
@@ -321,6 +429,119 @@ Please use the format 'my-cluster' or '${instance.name}/clusters/my-cluster'.`);
     );
   }
 
+  getBackups(options?: GetBackupsOptions): Promise<GetBackupsResponse>;
+  getBackups(options: GetBackupsOptions, callback: GetBackupsCallback): void;
+  getBackups(callback: GetBackupsCallback): void;
+  /**
+   * Get Cloud Bigtable Backup instances within this cluster. This returns both
+   * completed and pending backups.
+   *
+   * @param {GetBackupsOptions | GetBackupsCallback} [optionsOrCallback]
+   * @param {GetBackupsResponse} [callback] The callback function.
+   * @param {?error} callback.error An error returned while making this request.
+   * @param {Backup[]} callback.backups All matching Backup instances.
+   * @param {object} callback.apiResponse The full API response.
+   * @return {void | Promise<ListBackupsResponse>}
+   */
+  getBackups(
+    optionsOrCallback?: GetBackupsOptions | GetBackupsCallback,
+    cb?: GetBackupsCallback
+  ): void | Promise<GetBackupsResponse> {
+    const options =
+      typeof optionsOrCallback === 'object' ? optionsOrCallback : {};
+    const callback =
+      typeof optionsOrCallback === 'function' ? optionsOrCallback : cb!;
+
+    const reqOpts: google.bigtable.admin.v2.IListBackupsRequest = {
+      ...options,
+      parent: this.name,
+    };
+
+    delete (reqOpts as GetBackupsOptions).gaxOptions;
+
+    this.bigtable.request<google.bigtable.admin.v2.IBackup[]>(
+      {
+        client: 'BigtableTableAdminClient',
+        method: 'listBackups',
+        reqOpts,
+        gaxOpts: options.gaxOptions,
+      },
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      (err, ...resp: any[]) => {
```

Comment:
@AVaksman could you advise here? This was taken from your suggestion on the other PR. I'm fighting a lot of TypeScript errors when I tamper with this.

Reply:
I took a stab at it as a part of an existing PR #731 where I am differentiating `bigtable.request` signature between `NormalCallback`, `LROCallback` and `PaginatedCallback`
```ts
export type RequestCallback<T = void, R = void> = T extends void
  ? LROCallback
  : R extends void
  ? NormalCallback<T>
  : PagedCallback<T, R>;

export interface NormalCallback<TResponse> {
  (err: ServiceError | null, res?: TResponse): void;
}

export interface PagedCallback<Item, Response> {
  (
    err: ServiceError | null,
    results?: Item[] | null,
    nextQuery?: {} | null,
    response?: Response | null
  ): void;
}

export interface LROCallback {
  (
    err: ServiceError | null,
    operation?: Operation,
    apiResponse?: IOperation
  ): void;
}
```

Then here the something like this would work
```ts
this.bigtable.request<
      google.bigtable.admin.v2.IBackup,
      google.bigtable.admin.v2.IListBackupsResponse
    >(
      {
        client: 'BigtableInstanceAdminClient',
        method: 'listAppProfiles',
        reqOpts,
        gaxOpts,
      },
      (err, IBackups, nextPageRequest, apiResponse) => {
        if (err) {
          callback(err);
          return;
        }
        const backups = IBackups!.map(backup => {
          const backupInstance = this.backup(backup.name!.split('/').pop()!);
          backupInstance.metadata = backup;
          return backupInstance;
        });
        const nextQuery = nextPageRequest!
          ? extend({}, options, nextPageRequest!)
          : null;
        callback(null, appProfiles, nextQuery, apiResponse!);
      }
    );
```
Is this an overkill?
`bigtable.request` signature changes could be done in a separate PR in a non breaking way.
WDYT?

Reply:
If agree, I can do the `bigtable.request` PR?

Reply:
Curious for @JustinBeckwith's thoughts. Whatever makes sense to you guys :)

For now, should I leave the code as-is? Also, please LMK if everything else looks correct so far and I'll get the tests going.

Reply:
I am ok with leaving this as is for now, and coming back around to improve the under types here later.  I'm ok with it because I don't think they affect the public types.  Not a big deal.